scripts/git_pull_tul_cob.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 # Argument 1 is the LATEST_RELEASE flag, if "true" fetch latest tagged release
 # Argument 2, if arg 1 is not true, is any git ref or branch to switch to
+
+if [ ! -d $HOME/tul_cob ];
+then
+  git clone https://github.com/tulibraries/tul_cob.git $HOME/tul_cob
```

Comment:
1. is there any chance of conflicting branches of tul_cob being used by this DAG and the indexing DAG?